test/test.sh

```diff
@@ -93,6 +93,10 @@ fi
 
 $BIN version
 
+# Install genuinetools/reg
+sudo curl -fSL "https://github.com/genuinetools/reg/releases/download/v0.16.1/reg-linux-amd64" -o "/usr/local/bin/reg"
```

Comment:
Can we move this to: https://github.com/flanksource/commons/blob/master/deps/deps.go#L60